web-api/src/personalization/routes.rs

```diff
@@ -542,7 +543,7 @@ async fn semantic_search(
         documents: documents.into_iter().map_into().collect(),
     }));
 
-    tracing::info!(elapsed_time = request_start.elapsed().as_millis());
+    debug!(elapsed_time = request_start.elapsed().as_millis());
```

Comment:
This should stay `info` to avoid having to change the level of the whole application just to get some data from this. It will be removed.